### ISSUE TITLE
Error in twitter-list-lists.py

### ISSUE THREAD

**Issue by DenverDaveH** [ideoforms/python-twitter-examples#4]
Hello,
I am running python 2.7 and 3.5. I also have Anaconda3 installed. In any version I am getting this error:
/media/sf_Temp/python-twitter-examples$ python /media/sf_Temp/python-twitter-examples/twitter-list-lists.py 
Traceback (most recent call last):
  File "/media/sf_Temp/python-twitter-examples/twitter-list-lists.py", line 23, in <module>
    twitter = Twitter(auth=OAuth(config["access_key"], config["access_secret"], config["consumer_key"], config["consumer_secret"]))
NameError: name 'Twitter' is not defined

I cloned your examples to try and learn more about the Twitter api. So I want to run the scripts as is, before hacking around on them.

Any thoughts as to why this is throwing this error?
Thanks

**Comment by beatrizChagas** [comment]
Do you have a Twitter API installed? Which?

**Comment by DenverDaveH** [comment]
Yes I did a pip install twitter-python.

**Comment by beatrizChagas** [comment]
Try this  https://github.com/geduldig/TwitterAPI

**Comment by aljohaniAbdula** [comment]
same problem with "NameError: name 'Twitter' is not defined", have same specs

**Comment by beatrizChagas** [comment]
This error occurs when python interpreter do not recognize the importated library

**Comment by DenverDaveH** [comment]
Ok so as I have installed the api using the pip command, what else could it be? I cloned this repo. Tried to run python setup.py and get this error:
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: no commands supplied
This assumes if the pip install is not working, then this should do the install via setup.py?
